jung-algorithms/src/main/java/edu/uci/ics/jung/layout/algorithms/CircleLayoutAlgorithm.java

```diff
@@ -49,7 +51,7 @@ public void setRadius(double radius) {
 
   /**
    * Sets the order of the nodes in the layout according to the ordering specified by {@code
-   * comparator}.
+   * comparator}. An effective comparator could be used to reduce or minimize the crossing number
```

Comment:
I don't think that the comparator was specifically for trying to reduce edge crossings so much as just allowing the user to provide an easily understandable ordering (whether that's semantic grouping, or alphabetical, or whatever).

So I'd prefer not to include language about that, because as I dimly recall (it's been a long time since I looked up layout algorithms for this) avoiding edge crossings is actually a hard problem in layouts.